x/profiles/simulation/decoder.go

```diff
@@ -23,6 +23,11 @@ func DecodeStore(cdc *codec.Codec, kvA, kvB kv.Pair) string {
 		cdc.MustUnmarshalBinaryBare(kvA.Value, &addressA)
 		cdc.MustUnmarshalBinaryBare(kvB.Value, &addressB)
 		return fmt.Sprintf("AddressA: %s\nAddressB: %s\n", addressA, addressB)
+	case bytes.HasPrefix(kvA.Key, types.DTagTransferRequestPrefix):
```

Comment:
Since it looks like the `DTagTransferRequestPrefix` will represent a list of transfer requests, shouldn't it be more appropriate to rename it `DTagTransferRequestsPrefix`?